index.html

```diff
@@ -5,8 +5,33 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e2597287a1.js" crossorigin="anonymous"></script>
   <title>Portfolio</title>
 </head>
 <body>
+  <div class="app-cont">
+    <div class="header-cont">
+      <div class="nav-cont">
+        <div class="header-img"><img src="images/image_geometry_1.png" alt="">
+        </div>
+        <nav class="nav-bar">
+          <i class="fa-solid fa-bars fa-xl"></i>
+      </div>
+      <header>
+        <h1 class="header-h1">
+          Fuad <br> Nabiyev
+        </h1>
+        <p class="header-p">
+          Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+          If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+      </header>
+      <div class="start-btn">
+        <h2 class="btn-h2">
+          Start collaboration
+        </h2>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] it would be better if you add the illustration (design image) as in the Figma design in order to match and you should add it as background with CSS not as an image cause it's only for style
<img width="268" alt="Screen Shot 2022-09-21 at 17 40 56" src="https://user-images.githubusercontent.com/61790587/191565254-c6520037-2937-40c2-8390-64e4cdb834aa.png">
 it